maven-javadoc-plugin/pom.xml

```diff
@@ -148,7 +148,7 @@ under the License.
     <dependency>
       <groupId>org.apache.maven</groupId>
       <artifactId>maven-archiver</artifactId>
-      <version>2.5</version>
+      <version>2.6</version>
```

Comment:
Drop this change. If you want to upate maven archiver, create a separate JIRA ticket for that.